src/message/headers/StreamMessageHeader.js

```diff
@@ -17,7 +17,6 @@ const styles = StyleSheet.create({
     alignItems: 'center',
     flexDirection: 'row',
     backgroundColor: '#eee',
-    overflow: 'hidden',
```

Comment:
Ughhh. It seems not. Though in 10 times tested it did?!?
That is better though, then the issue is something else.